ftw/book/tests/test_table_export_import.py

```diff
@@ -29,6 +34,37 @@ def test_export_table(self, browser):
             'text/csv; charset=utf-8',
             browser.headers.get('content-type'))
 
+    @browsing
+    def test_non_ascii_export(self, browser):
+        table2 = create(Builder('table')
+                        .titled(u'Population')
+                        .with_table((('Ranking', 'City', 'Population'),
+                                     ('0', u'Gu\xe4ngzhou', '44 mil <sup>1</sup>')))
+                        .having(border_layout='grid',
+                                footnote_text=RichTextValue(
+                                    u'<p><sup>1</sup> thats quite big</p>'))
+                        .within(self.table.aq_parent))
+        browser.login().visit(table2, view='table_export_import')
+        try:
+            browser.click_on('Export')
+        except HTTPServerError:
```

Comment:
Please catch Exception 
(suggest adding a comment saying "probably HTTPServerError)

Reply:
As discussed since the test above tests the positive aspect and any other exception will simply error the test - I'm persuaded this is reasonable - well argued!